### PR TITLE
#428 update linux build instruction

### PR DESCRIPTION
**Pull request by savoptik**
Hello. I recently ran into [this](https://github.com/RHVoice/RHVoice/issues/428#issuecomment-1002266719) problem. I suggest updating the assembly instructions to avoid problems in the future.

### CODE REVIEW

File: doc/en/Compiling-on-Linux.md [hunk 1]
```diff
@@ -21,6 +21,19 @@ Please note that many distributions provide a separate development package for
 a library, which is necessary to compile the programs using the library.
 You need to install both runtime and development packages.
 
+## Get sources
+
+1. One line
+    ```bash
+    git clone --recursive https://github.com/RHVoice/RHVoice.git
+    ```
+2. Stap by stap
```
Comment (alex19EP) [inline doc/en/Compiling-on-Linux.md:30]:
```suggestion
2. Step by step
```

File: doc/en/Compiling-on-Linux.md [hunk 2]
```diff
@@ -40,7 +53,7 @@ scons -h
 To install RHVoice execute the following command:
 
 ```bash
-# scons install
+sudo scons install
```
Comment (alex19EP) [inline doc/en/Compiling-on-Linux.md:56]: someone may not use sudo I would like to leave this as before.

File: doc/ru/Compiling-on-Linux.md [hunk 3]
```diff
@@ -45,7 +58,7 @@ scons -h
 Для установки RHVoice выполните следующую команду:
 
 ```bash
-# scons install
```
Comment (alex19EP) [inline doc/ru/Compiling-on-Linux.md:48]: см. выше

**Review by alex19EP — CHANGES_REQUESTED** [review]

File: doc/en/Compiling-on-Linux.md (hunk 2 above)
Comment (savoptik) [reply doc/en/Compiling-on-Linux.md:56]: done

File: doc/ru/Compiling-on-Linux.md (hunk 3 above)
Comment (savoptik) [reply doc/ru/Compiling-on-Linux.md:48]: Done

File: doc/en/Compiling-on-Linux.md (hunk 1 above)
Comment (savoptik) [reply doc/en/Compiling-on-Linux.md]: done

**Review by alex19EP — APPROVED** [review]

**Pull request merged**